packages/examples/.storybook/preview.js

```diff
@@ -3,10 +3,20 @@ import { initializeIcons } from '@uifabric/icons';
 import { configure, addParameters, addDecorator } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
 import { withA11y } from '@storybook/addon-a11y';
+import { withPerformance } from 'storybook-addon-performance';
```

Comment:
These changes are copied from updates to the OUFR storybook config.

All individual packages' storybook helpers/config should be deleted as part of the final PR. The generateStoriesFromExamples helper can also be deleted.